src/arrays/programs_involving_multidimensional_data/hard/q1/Sums.java

```diff
@@ -0,0 +1,60 @@
+package arrays.programs_involving_multidimensional_data.hard.q1;
+
+/*
+Write a Java program called Sums. It should have a main method which takes user input to create a 2D array and calls
```

Comment:
Change question text to:
`Write a Java program called Sums. It should have a main method which takes user input to create a 2D integer array and calls another method called sums. The sums method should take a 2D integer array as a parameter and return a 1D integer array which is the sum of each row in the 2D array. The main method should then print the 1D array.`